src/main/java/org/craftercms/studio/impl/v1/deployment/PreviewDeployer.java

```diff
@@ -207,12 +207,16 @@ public void syncAllContentToPreview(RepositoryEventMessage message) throws Servi
     protected void syncFolder(String site, String path) {
         RepositoryItem[] children = contentRepository.getContentChildren(site, path, true);
 
-        for (RepositoryItem item : children) {
-            if (item.isFolder) {
-                syncFolder(site, item.path + "/" + item.name);
-            } else {
-                deployFile(site, contentService.getRelativeSitePath(site, item.path + "/" + item.name));
+        if (children != null) {
+            for (RepositoryItem item : children) {
+                if (item.isFolder) {
+                    syncFolder(site, item.path + "/" + item.name);
```

Comment:
We should create a Util called JoinPaths or something and get this  + "/" + junk out of our code.  and that util should correct for "//" or "///" or missing "/" scenarios